### DISCUSSION TITLE
Shell script that output helix commands

### DISCUSSION THREAD

**Discussion by tdaron** [helix-editor/helix#8990] [Ideas] [answered]
Hi everyone ! 

While using helix, i heavily rely on the `:sh` command, allowing me to execute arbitrary script directly from helix, which is great ! 

But i recently though about a command that would run a shell script, that outputs other helix commands that will be executed.

Example, i could make a `debug.sh` script inside my project repo, that would output different helix command to start debug with the good target, or whatever ! It would be IMO a good way to improve helix flexibility while staying lightweight.

What are your thoughts about this ?

**Comment by the-mikedavis** [comment, marked as answer]
Duplicate of https://github.com/helix-editor/helix/discussions/8944
